src/components/MdTable/MdTablePagination.vue

```diff
@@ -4,19 +4,19 @@
       <span class="md-table-pagination-label">{{ mdLabel }}</span>
 
       <md-field>
-        <md-select v-model="currentPageSize" md-dense md-class="md-pagination-select" @changed="setPageSize">
+        <md-select v-model="currentPageSize" md-dense md-class="md-pagination-select" @md-selected="setPageSize">
           <md-option v-for="amount in mdPageOptions" :key="amount" :value="amount">{{ amount }}</md-option>
         </md-select>
       </md-field>
     </template>
 
     <span>{{ currentItemCount }}-{{ currentPageCount }} {{ mdSeparator }} {{ mdTotal }}</span>
 
-    <md-button class="md-icon-button md-table-pagination-previous" @click="goToPrevious()" :disabled="mdPage === 1">
+    <md-button class="md-icon-button md-table-pagination-previous" @click="goToPrevious()" :disabled="currentPage === 1">
       <md-icon>keyboard_arrow_left</md-icon>
     </md-button>
 
-    <md-button class="md-icon-button md-table-pagination-next" @click="goToNext()">
+    <md-button class="md-icon-button md-table-pagination-next" @click="goToNext()" :disabled="currentPageCount === mdTotal">
```

Comment:
`:disabled="currentPageCount === mdTotal"`
this is not correct, if I show 20 items per page and have total 35 items then on second page `currentPageCount` will be equal 40 and `mdTotal` - 35

Reply:
I will take a look at this and push a fix. thank you.

Reply:
I have just pushed a fix for this. Realised I had been passing through the wrong value into total so that I could never experience this bug.